spaces/S000107/properties/P000008.md

```diff
@@ -2,10 +2,10 @@
 uid: T001134
 space: S000107
 property: P000008
-value: true
+value: false
 refs:
-- doi: 10.1007/978-1-4612-6290-9_6
-  name: Counterexamples in Topology
+- doi: 10.4064/fm-88-2-127-132
+  name: VAN DOWEN, Eric; The box product of countably many metrizable spaces need not be normal
```

Comment:
Typo, and would prefer not all caps.  It's actually "van Douwen, Eric" (with a lowercase "v", it's Dutch.  See the pdf)

Reply:
Done